### PR TITLE
MULTISITE-16209 - Hotfix - files_temporary_path in the settings generation

### PR DESCRIPTION
**Pull request by verbruggenalex**

### CODE REVIEW

File: build.properties.dist [hunk 1]
```diff
@@ -160,6 +160,7 @@ subsite.temporary.development.make = ${phing.project.tmp.dir}/development.make
 # Files and directories inside the Drupal installation for local development.
 platform.build.settings.dir = ${platform.build.sites.dir}/default
 platform.build.files.dir = ${platform.build.settings.dir}/files
+platform.build.files.dir.tmp = ${platform.build.files.dir}/tmp
```
Comment (jonhy81) [inline build.properties.dist]: Hi @verbruggenalex , this should be ```platform.build.files.dir.tmp = ${platform.build.settings.dir}/tmp```, or is intentional to create the folder ```tmp``` inside ```files```.

File: build.clone.xml [hunk 2]
```diff
@@ -144,8 +144,7 @@
                     <token key="theme_default" value="${development.variables.theme_default}" />
                     <token key="file_public_path" value="${platform.build.files.dir}" />
                     <token key="file_private_path" value="${platform.build.files.dir}/private_files" />
-                    <!-- Still thinking on wether a new property is needed for this. -->
-                    <token key="file_temporary_path" value="@todo" />
+                    <token key="file_temporary_path" value="${platform.build.files.dir.tmp}" />
```
Comment (jonhy81) [inline build.clone.xml]: ```<token key="file_temporary_path" value="${platform.build.tmp.dir}" />```

File: build.package.xml [hunk 3]
```diff
@@ -387,6 +387,7 @@
             <istrue value="${platform.build.files.dir}" />
             <then>
                 <mkdir dir="${platform.build.files.dir}/private_files" />
+                <mkdir dir="${platform.build.files.dir.tmp}" />
```
Comment (jonhy81) [inline build.package.xml]: ```<mkdir dir="${platform.build.tmp.dir}" />```

**Comment by verbruggenalex** [comment]
:+1:

**Pull request merged**